x-pack/plugins/case/server/routes/api/utils.test.ts

```diff
@@ -222,7 +222,12 @@ describe('Utils', () => {
       ];
 
       const res = transformCases(
-        { saved_objects: mockCases, total: mockCases.length, per_page: 10, page: 1 },
+        {
+          saved_objects: mockCases.map((obj) => ({ ...obj, score: 1 })),
```

Comment:
`mockCases` are used to test all repository methods. As the type from `find` now differs from the other, the simplest fix that to add the score in the associated tests.

Reply:
so you're saying you only are adding the score to certain methods, and thats why you don't just add it to the `mockCases` object directly?

Reply:
That's right, the es `score` is only returned from the `find` SO API. Other apis such as `get`, `bulkGet` and so remains unchanged.